Testscripts/Windows/NESTED-HYPERV-STORAGE-PERF.ps1

```diff
@@ -2,6 +2,48 @@
 # Licensed under the Apache License.
 param([object] $AllVmData, [object] $CurrentTestData)
 
+# For HyperV, disable the autoconfiguration IPv4 to avoid getting a private IP from the VM.
+function Disable-APIPA ($session) {
```

Comment:
AFAIK you'll end up with APIPA only if there is a communication problem with the dhcp server - like the IP pool is full and cannot get an IP lease.
But DHCP will always be used first, and APIPA is more of a fallback solution.
Are you sure you're looking at the core problem here and not creating some workaround?

Reply:
@chvalean Appreciate for your review.
This is for nested VM testing, L1 is windows (without dhcp), L2 is Linux, we will get 2 IPs from the L2 VM, one starts with 169 (we also can ping this IP), the other is a domain IP.
We are trying two ways 
1 inject IP from host side
2 install dhcp server in L1 and set IP range

Any idea?

Reply:
I see.
would definitely go with injecting IP, setting up a dhcp server might cause other issues or prone to conflicts in the network.